src/Graphex/Logger.hs

```diff
@@ -0,0 +1,19 @@
+module Graphex.Logger (logit) where
```

Comment:
I didn't commit anything in this module, but I did use it for testing.

It's just a wrapper around `Debug.Trace` with a verbosity controlled by an env var.